src/duplo_resource/service.py

```diff
@@ -34,6 +34,27 @@ def find(self,
     except IndexError:
       raise DuploError(f"Service '{name}' not found", 404)
 
+  @Command()
+  def update_replicas(self, name: args.NAME,
+                      replica: args.REPLICAS):
+    """Update number of replicas for a service.
+
+    Args:
+        name (str): The name of the service to update.
+        replica (str): Number of replicas to set for service.
+    """
+    try:
+      tenant_id = self.tenant["TenantId"]
+      data = {
+        "Replicas": replica,
+        "Name": name
+      }
+      response = self.duplo.post(f"subscriptions/{tenant_id}/ReplicationControllerChangeAll", data)
+      if response.json() is None:
+        return {"message": f"Successfully updated replicas for service '{name}'"} 
+    except IndexError:
```

Comment:
How does this happen? I wouldn't expect to see this at this level, something below should catch these and then rethrowing as DuploError

Reply:
Api gives null response on successful post call. So i added a condition